src/main/kotlin/no/nav/helse/flex/domain/KafkaMeldinger.kt

```diff
@@ -10,7 +10,11 @@ data class MeldingMedEventName(
 )
 
 fun String.hentEventName(): String? {
-    return objectMapper.readValue(this, MeldingMedEventName::class.java).eventName
+    return try {
+        objectMapper.readValue(this, MeldingMedEventName::class.java).eventName
+    } catch (e: Exception) {
+        null
```

Comment:
`log.warn("Noe fornuftig")` ?

Reply:
la logginga i metoden som kalte, for å få med partiotion og offset